### PR TITLE
Image Type Proposal

### PR DESCRIPTION
**Pull request by jimas14**
This PR includes the proposal for adding the trait `image_type` for search/UI display purposes in Anglerfish.

### CODE REVIEW

**Comment by jpblair** [comment]
@RyanGladstone Do we plan to do anything with this in WebSked?

**Comment by RyanGladstone** [comment]
Joseph, there are no plans to add image-related features within WebSked.

File: docs/proposals/2018-12-18 - Image Type Proposal.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+```
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.8.1/traits/trait_image_type.json",
+  "title": "Image Type",
+  "description": "A more specific category for an image.",
+  "type": "string",
+  "category": {
+    "type": "string",
+    "enum": [ "photograph", "graphic", "illustration", "thumbnail" ],
```
Comment (anscho) [inline docs/proposals/2018-12-18 - Image Type Proposal.md]: If an image doesn't fit a category – though I'm not sure what that would be – will that element field not include this trait?

Comment (jimas14) [reply docs/proposals/2018-12-18 - Image Type Proposal.md]: Yup, the field will just not exist. For search purposes, they'll be considered Photographs though. That way all old content will still appear in searches by image_type, when type is Photograph.

File: docs/proposals/2018-12-18 - Image Type Proposal.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+## Illustration
+
+### Definition
+
+A creative work made for expression.
+
+### Rights and Usage
+
+Most infographics are owned by the customer. If owned by the customer, indefinitely. If not, third-party is usually not generous.
```
[the remark this answers is not in the document]
Comment (jimas14) [reply docs/proposals/2018-12-18 - Image Type Proposal.md]: Indeed it should! Thanks

**Comment by jimas14** [comment]
Force pushed to remove commits that added fixtures and validation. I realized that wasn't necessary since this is a trait and will be validated by the existing `image-fixture-good.json` that now has an `image_type` property.

**Pull request merged**